components/event-simulator/org.wso2.carbon.event.simulator.admin/src/main/java/org/wso2/carbon/event/simulator/admin/EventSimulatorAdminService.java

```diff
@@ -334,6 +338,7 @@ public boolean saveDataSourceConfigDetails(String dataSourceConfigAndEventStream
                     DataSourceTableAndStreamInfoDtoArray[index].setStreamAtrributeNames(dataSourceTableAndStreamInfo.getDataSourceColumnsAndTypes()[1]);
                     DataSourceTableAndStreamInfoDtoArray[index].setFileName(dataSourceTableAndStreamInfo.getFileName());
                     DataSourceTableAndStreamInfoDtoArray[index].setFilePath(dataSourceTableAndStreamInfo.getFilePath());
+                    DataSourceTableAndStreamInfoDtoArray[index].setDelayBetweenEventsInMilies(dataSourceTableAndStreamInfo.getDelayBetweenEventsInMilies());
```

Comment:
Thanks tharik, those formatting is already been made in commit - https://github.com/ramindu90/carbon-event-processing/commit/bdc2e693bad901f641cc52a5274f01d13aed2592